kyo-core/js/src/main/scala/kyo/KyoAppPlatformSpecific.scala

```diff
@@ -0,0 +1,30 @@
+package kyo
+
+import kyo.Maybe.Absent
+import kyo.Maybe.Present
+
+abstract class KyoAppPlatformSpecific extends KyoApp.Base[Async & Resource & Abort[Throwable]]:
+
+    private var maybePreviousAsync: Maybe[Unit < (Async & Abort[Throwable])] = Absent
+
+    final override protected def run[A: Flat](v: => A < (Async & Resource & Abort[Throwable]))(using Frame): Unit =
+        import AllowUnsafe.embrace.danger
+        val currentAsync: Unit < (Async & Abort[Throwable]) =
+            Async.timeout(timeout)(handle(v)).map(value => printResult(Result.success(value)))
+        maybePreviousAsync = maybePreviousAsync match
+            case Absent                 => Present(currentAsync)
+            case Present(previousAsync) => Present(previousAsync.map(_ => currentAsync))
+        if initCode.isEmpty then
```

Comment:
I was initially wondering **why** this would work, when only the first fiber would be initialized, but then I realized the first fiber would begin the execution of the next...

In my testing, I think this code also causes `repeatWithDelay` to run forever. The `run` fiber is already complete when the `timeout` expires.

Regardless, I re-wrote this a bit which I think is a simplification:

```scala
package kyo

import kyo.Maybe.Absent
import kyo.Maybe.Present

abstract class KyoAppPlatformSpecific extends KyoApp.Base[Async & Resource & Abort[Throwable]]:

    private var previous: Maybe[Unit < (Async & Abort[Throwable])] = Absent

    final override protected def run[A: Flat](v: => A < (Async & Resource & Abort[Throwable]))(using Frame): Unit =
        import AllowUnsafe.embrace.danger
        val current: Unit < (Async & Abort[Throwable]) =
            Abort.run(handle(v)).map(result => IO(printResult(result)).andThen(Abort.get(result)).unit)

        previous = previous match
            case Absent            => Present(current)
            case Present(p) => Present(p.map(_ => current))

        initCode = previous.map { async => () =>
            val raced = Clock.repeatWithDelay(1.hour)(()).map { fiber =>
                val race = Async.race(fiber.get, async)
                Async.timeout(timeout)(race)
            }
            val _ = IO.Unsafe.run(Async.run(raced)).eval
        }.toList

    end run

end KyoAppPlatformSpecific
```

In `KyoApp` I made `initCode` a variable with a List for ease of conversion with Maybe.
```scala
protected var initCode: List[() => Unit] = List.empty
```

You can check if the Clock fiber is still running by changing `()` to a `Console.println` and increase the frequency of heartbeat.